src/Azure.Functions.Cli/Common/AppSettingsFile.cs

```diff
@@ -29,7 +29,7 @@ public AppSettingsFile(string filePath)
             {
                 Values = new Dictionary<string, string>();
                 ConnectionStrings = new Dictionary<string, JToken>();
-                IsEncrypted = true;
+                IsEncrypted = false;
```

Comment:
Why is this being changed?

Reply:
@liliankasem Changed it because of the below warning:
![image](https://github.com/user-attachments/assets/7c5bb07d-5407-496a-b8b6-5637222d3559)
the value of setting (FUNCTION_WORKER_RUNTIME) is encrypted.

Reply:
I think this is something you can do to decrypt it? I don't think we should be changing this setting

@fabiocav fyi

Reply:
@liliankasem Yes, we can decrypt it and there is no issue with that. it will work in either way

Reply:
Great, then let's do that and keep `IsEncrypted` = true